### PR TITLE
Canister behaviour with pipes

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Adds a sprite overlay for securing a canister to a connector
Blocks canisters from being pushed or dragged when connected.
Pipes will now change its sorting layer when secured or unsecured to the floor.
image

### Open Questions and Pre-Merge TODOs

- [X]  This fix is tested on the same branch it is PR'ed to.
- [ ]  I correctly commented my code
- [X]  My code is indented with tabs and not spaces
- [X]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [X]  This PR does not bring up any new compile errors
- [X]  This PR has been tested in editor
- [ ]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs [hunk 1]
```diff
@@ -33,6 +33,7 @@ public void WrenchAct()
 		if (anchored)
 		{
 			anchored = false;
+			spriteRenderer.sortingLayerID = SortingLayer.NameToID("Items");
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs:36]: you could cache these and make static, but it’s a minor optimisation

**Review by fomalsd — APPROVED** [review]
Nice!

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs (hunk 1 above)
Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs:36]: yeah I kinda want to copy all layers from tg too

Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs:36]: actually just an idea because I dunno how they are defined

**Pull request merged**